[Terraform] Fix examples (#8911)
cloudflare/cloudflare-docs: content/terraform/tutorial/add-page-rules.md

      + actions {
          + always_use_https    = false
          + disable_apps        = false
          + disable_performance = false
          + disable_railgun     = false
          + disable_security    = false
          + disable_zaraz       = false
          + security_level      = "under_attack"
        }
    }

  # cloudflare_page_rule.redirect-to-new-db-page will be created
  + resource "cloudflare_page_rule" "redirect-to-new-db-page" {
      + id       = (known after apply)
      + priority = 2
      + status   = "active"
      + target   = "www.example.com/old-location.php"
      + zone_id  = "e2e6491340be87a3726f91fc4148b126"

      + actions {
          + always_use_https    = false
          + disable_apps        = false
          + disable_performance = false
          + disable_railgun     = false
          + disable_security    = false
          + disable_zaraz       = false

          + forwarding_url {
              + status_code = 301
              + url         = "https://www.example.com/expensive-db-call"
            }
        }
    }

Plan: 2 to add, 0 to change, 0 to destroy.

------------------------------------------------------------------------

> Note: You didn't use the -out option to save this plan, so Terraform can't
> guarantee to take exactly these actions if you run "terraform apply" now.

Reviewer: ```suggestion
Note: You did not use the `-out` option to save this plan, so Terraform cannot guarantee to take exactly these actions if you run `terraform apply` now.
```

Author: (Also part of the tool output.)
